MAINTAINERS

```diff
@@ -1,6 +1,5 @@
 The maintainers in alphabetical order are:
 
-Chris Sanders, Microsoft <chris.sanders@microsoft.com> (github: csand-msft)
 Chris Short, Amazon Web Services <cbshort@amazon.com> (github: chris-short)
 Christian Hernandez, <christian@chernand.io> (GitHub: christianh814)
 Dan Garfield, Codefresh <dan@codefresh.io> (github: todaywasawesome)
```

Comment:
Code suggestion for my last comment. We may as well add Chris S and Chris P too since #119 is already merged and we're adding a new Emeritus section in this PR:

```suggestion
Dan Garfield, Codefresh <dan@codefresh.io> (github: todaywasawesome)

Emeritus:

Chris Sanders, Microsoft <chris.sanders@microsoft.com> (github: csand-msft)
Chris Patterson, GitHub <chrispat@github.com> (github: chrispat)
```